src/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Created by chetanv on 07/06/16.
+ */
+
+import FactoryGirl from './FactoryGirl';
+
+const factory = new FactoryGirl();
+factory.FactoryGirl = FactoryGirl;
+
+export default factory;
+
+// TODO: Object adapter still works with models. Either change that or,
+//       add a new JSONAdapter, or maybe we don't need it at all as we now have
+//       assocAttrs
```

Comment:
While I wrote ObjectAdapter, I misunderstood it as something that work with Model classes but doesn't persist it. I later realised that that's not the case, and in current version it is used to just build and embed attrs. By that time I had already added AssocAttrs as well, hence the comment.